### DISCUSSION TITLE
Allow ingress traffic for slack slash command

### DISCUSSION THREAD

**Discussion by animer3009** [outline/outline#4858] [Self Hosting]
Hi,
We are running Outline in our GCP.
External access is closed. Just open for a few IP's.
Server/Services access we control using GCP Firewall.

We need to make work a Slack slash command /outline .
When we open access for 0.0.0.0/0 it works fine. We can't open access only for Slack, as it has no static IP ranges...

I guess the only solution is to make something like a proxy.
Any idea or any experience/tutorial?

**Comment by alxjsn** [comment]
I was interested in this as well. Maybe adding support for Socket Mode `https://api.slack.com/methods/apps.connections.open` would fix this?

Reference: https://app.slack.com/app-settings/T023GPQFACF/A04R4GEMHV5/socket-mode
